lib/Twig/Environment.php

```diff
@@ -1280,28 +1288,10 @@ protected function initExtension(Twig_ExtensionInterface $extension)
 
     protected function writeCacheFile($file, $content)
```

Comment:
This method is currently not called anymore which is not BC. So I guess we still need to call it from loadTemplate but disable the deprecation in this case (via additional optional argument).